CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Help/DataSources/HelpBaseDataSource.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  HelpBaseDataSource.swift
+//  
+//
+//  Created by Krzysztof Kryniecki on 02/08/2022.
+//  Copyright © 2022 Gini GmbH. All rights reserved.
+//
+
+import UIKit
+
+public class HelpBaseDataSource<Item, Cell>: NSObject, UITableViewDelegate, UITableViewDataSource  where Cell: HelpCell {
+    
+    var items: [Item] = []
+    let giniConfiguration: GiniConfiguration
+    
+    init(
+        configuration: GiniConfiguration
+    ) {
+        giniConfiguration = configuration
+    }
+    
+    public func configureCell(cell: Cell, indexPath: IndexPath) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    @objc public func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return items.count
+    }
+    
+    @objc(tableView:cellForRowAtIndexPath:) public func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        if let cell = tableView.dequeueReusableCell(withIdentifier: Cell.reuseIdentifier) as? Cell {
+            self.configureCell(cell: cell, indexPath: indexPath)
+            return cell
+        }
+        fatalError()
+    }
+    
+    public func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+    
+    @objc(tableView:willDisplayCell:forRowAtIndexPath:) public func tableView(_ tableView: UITableView, willDisplay cell: UITableViewCell, forRowAt indexPath: IndexPath) {
```

Comment:
Is it a new style how to add @objc? I've never seen it.

Reply:
Good catch! hmm ... It was automatically added by Xcode after some complains. I need to check it maybe it's no longer necessary

Reply:
It seems so, but it's no longer needed it seems redundant. I will remove it